### PR TITLE
docs(style): fix json code block syntax

### PR DESCRIPTION
**Pull request by soulbird**
1、The tag of the json code block in markdown should be 'json' instead of 'shell'
2、Adjusting file alignment to make the document more formal

### CODE REVIEW

File: docs/zh/latest/admin-api.md [hunk 1]
```diff
@@ -598,7 +598,7 @@ APISIX 的 Upstream 除了基本的负载均衡算法选择外，还支持对上
 
 **upstream 对象 json 配置内容：**
 
-```shell
+```json
```
Comment (spacewander) [inline docs/zh/latest/admin-api.md:601]: Could you also update the English doc?

Comment (soulbird) [reply docs/zh/latest/admin-api.md:601]: done

Comment (membphis) [inline docs/zh/latest/admin-api.md:601]: we should use `shell` here.

image

File: docs/zh/latest/admin-api.md [hunk 2]
```diff
@@ -330,15 +330,15 @@ HTTP/1.1 200 OK
...
-    "upstream_id": "1",     # upstream 对象在 etcd 中的 id ，建议使用此值
-    "upstream": {},         # upstream 信息对象，不建议使用
-    "name": "测试svc",  # service 名称
-    "desc": "hello world",  # service 描述
-    "enable_websocket": true, #启动 websocket 功能
+    "id": "1",                # id
+    "plugins": {},            # 指定 service 绑定的插件
+    "upstream_id": "1",       # upstream 对象在 etcd 中的 id ，建议使用此值
+    "upstream": {},           # upstream 信息对象，不建议使用
+    "name": "测试svc",         # service 名称
```
Comment (spacewander) [inline docs/zh/latest/admin-api.md:339]: bad indent

**Comment by soulbird** [comment]
It contains Chinese, it seems that only. Is it possible to change '测试' to 'test'?

**Comment by leslie-tsang** [comment]
> It contains Chinese, it seems that only. Is it possible to change '测试' to 'test'?

Yes, LTGM

**Review by spacewander — APPROVED** [review]

**Review by tzssangglass — APPROVED** [review]

**Pull request merged**